.github/workflows/gcp_deploy_prod.yaml

```diff
@@ -0,0 +1,31 @@
+name: Deploy to GCS
+
+on:
+  push:
+    tags:
+      - "20[0-9][0-9].[0-9][0-9].[0-9][0-9](-[0-9]+)?"
+  workflow_dispatch:
```

Comment:
Do we need this?

Reply:
not in the longer term, workflow dispatch just allows me to run the workflow manually without waiting on the pushed git tag.  So i can build a prod website during testing without creating multiple tags that will trigger the existing pipeline.